lib/chainweb/pact/request.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Kadena.Chainweb.Pact.Request do
+  @moduledoc """
+  Requests specification for `Chainweb.Pact` contracts.
+  """
+
+  alias Kadena.Types.{
+    ListenRequestBody,
+    ListenResponse,
+    LocalRequestBody,
+    LocalResponse,
+    PollRequestBody,
+    PollResponse,
+    SendRequestBody,
+    SendResponse,
+    SPVRequestBody,
+    SPVResponse
+  }
+
+  @type network :: :public | :test | nil
```

Comment:
It's `nil` a network type valid for Kadena? please remove

Reply:
srry my bad, fixed now